app/src/main/java/org/oppia/app/option/AppLanguageActivity.kt

```diff
@@ -0,0 +1,38 @@
+package org.oppia.app.option
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+private const val PREFERENCE_KEY ="PREFERENCE_KEY"
```

Comment:
It should actually be something more explanatory, like `APP_LANGUAGE_PREFERENCE_KEY`.

Reply:
Also, add space after `=`

Reply:
Done.